index.html

```diff
@@ -0,0 +1,342 @@
+        <li class="tonic dot"></li>
+        <li class="tonic back">Back End Dev</li>
+         <li class="tonic dot"></li>
+         <li class="tonic year">2015</li>
+       </ul>
+
+       <p class="paragraph2">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+     
+       <ul class="resources">
+        <li><a  class="button" href="#">HTML</a> <li>
+        <li><a  class="button" href="#">CSS</a> </li>
+        <li><a  class="button" href="#">javaScript</a></li>      
+      </ul>
+
+        <div class="action"> 
+          <form class="normalbutton">
+            <a class="seeproject"  href="#">See Project</a> 
+          </form> 
+      </div> 
+       
+      </div>
+    </div>
+  </section>
+ 
+  <!--Page 6-->
+<article class="aboutme">
+  
+  <section class="top">
+      <h2 id="heading">About <br>Myself</h2>
+      <p id="top paragraph">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      
+      <div class="socialmedia">
+
+      <h4 id="secondheading5">LET’S CONNECT</h4>
+      
+        <ul class="frameicons">
+         <li><a href="#"><img id="twitter"  src="sources/Vector4.png"></a></li>    
+         <li><a href="#"><img id="linkedin" src="sources/linkedin.png"></a></li>     
+         <li><a href="#"><img id="medium"   src="sources/Vector5.png"></a></li>        
+         <li><a href="#"><img id="github"  src="sources/Vector1.png"></a></li>        
+         <li><a href="#"><img id="peace"  src="sources/Vector3.png"></a></li>   
+        </ul>
+      </div>  
+
+  </div>
+  
+   
+    <form class="action">
+      <a class="normalbutton2"  href="#">Get my resume</a> 
+    </form>
+    
+  </section>
+
+<section class="skillslist">
+ <div class="frame12">
+
+  <section class="frame7">  
+    
+    <!--<img id="languages-separator" src="/sources/Languages Separator .png" alt="seperator">-->
+    
+    <img id="languages-dropdown" src="/MyPortfolio/sources/LanguagesChevron.png" alt="dropdown"> 
+
+  </section>
+
+  
+  <ul class="languagelist">
+      
+      <li class="languagelist jsbox"> 
+        <h7 class="jstext">JavaScript</h7>
+        <img class="jsimage" src="/MyPortfolio/sources/js.png" alt=""></li>
+        
+
+      <li class="languagelist htmlbox"> 
+        <h7 class="htmltext">HTML</h7>
+        <img class="htmlimage" src="/MyPortfolio/sources/s.png" alt=""></li>
+      
+      <li class="languagelist cssbox"> 
+        <h7 class="csstext">CSS</h7>
+        <img class="cssimage" src="/MyPortfolio/sources/css.png" alt=""></li>   
+        
+  </ul>
+ </div>
+ 
+ <ul>
+  <li id="languages"><h5>Languages</h5></li>
+ 
+  <li class="frame8">
+        <img class="chevron1" src="/MyPortfolio/sources/Chevron1.png" alt="">
+        <h4 class="frameworks">Frameworks</h4>
+  </li>
```

Comment:
- I guess you may have mistakenly added the image links wrongly because the images are currently not being rendered on the browser. 
- I believe you can refactor it by removing the `/MyPortfolio` added to it. Something like this 👉🏽 `src="./sources/Chevron1.png"`

Here is what i can see compared to Figma 👇🏽 

| Your Design  |Figma Design |
| :---:   | :---: | 
| <img width="378" alt="Screenshot 2022-12-08 at 12 40 08" src="https://user-images.githubusercontent.com/99246359/206441431-3bec53dd-b9cb-4759-8349-8049c02d6440.png"> |<img width="370" alt="Screenshot 2022-12-08 at 12 44 54" src="https://user-images.githubusercontent.com/99246359/206441469-657f6384-e8a4-41a1-a553-4dfcb715fbe1.png">  |

⚠️ Kindly ensure to apply this to all images in your "about me" section.